lib/Dialect/TritonGPU/Transforms/Pipeliner/MatmulLoopPipeline.cpp

```diff
@@ -115,12 +115,16 @@ createAsyncCopy(scf::ForOp &forOp, tt::LoadOp loadOp, Value alloc,
     alloc.replaceAllUsesWith(viewLoad.getResult());
     alloc.erase();
   } else {
+    SmallVector<ttg::LocalAllocOp> allocs_to_erase;
```

Comment:
nit: `allocsToErase` to follow MLIR stlye

Reply:
thanks